### ISSUE TITLE
StackedBarChart does not allow for Alt text to be defined

### ISSUE THREAD

**Issue by emokeeff** [microsoft/fluentui#15276]
### Environment Information

- **Package version(s)**: 3.1.5
- **Browser and OS versions**: 
OS build: 2004 (19041.508)
Edge Version: Version 85.0.564.63 (Official build) (64-bit)
URL: https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/charting/dist/index.html#/examples/stackedbarchart#Overview

### Describe the issue:

There is no ability to define an alt text to the Stacked Bar chart so all charts behave the same as the examples when using narrator.
[StackedBarChartExampleTrimmed.zip](https://github.com/microsoft/fluentui/files/5292316/StackedBarChartExampleTrimmed.zip)

### Please provide a reproduction of the issue in a codepen:

Can repro using examples of Stacked bar chart

1. Open URL https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/charting/dist/index.html#/examples/stackedbarchart#Overview
3. Navigate to any Stacked bar chart image using 'caps+right' arrow key and listen to narrator announcement.

#### Actual behavior:

On navigating to chart image using 'caps+right' key, Screen reader is only announcing as image. It is not announcing alt text for that image. There is no alt-text prop defined for Stacked Bar Chart so no alt-text can be added.

#### Expected behavior:

Alt text should be defined for all the non text content. This could be done if an alt-text is added to the Stacked Bar Chart Component

### Documentation describing expected behavior

User Impact:
Visually Impaired users who rely on screen readers will not be able to know about the image when narrator is not announcing alt text for the image.

1.1.1 - Non-text Content - https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html

**Comment by paulgildea** [comment]
@Raghurk FYI
